UnityProject/Assets/Scripts/UI/Core/WindowDrag.cs

```diff
@@ -56,8 +56,6 @@ public void UpdateMe()
 	private void OnDisable ()
 	{
 		UpdateManager.Remove(CallbackType.UPDATE, UpdateMe);
-
-		transform.localPosition = startPositon;
```

Comment:
I like adding functionality by removing code

Reply:
haha, well I found it tended to be annoying when doing chemistry since I would align the windows all laid out on my screen then walk away and they would be in the centre again